src/analysis/bounds.rs

```diff
@@ -275,7 +275,7 @@ impl Bounds {
         for used in &self.used {
             match used.bound_type {
                 NoWrapper => (),
-                IsA(_) => imports.add("glib::object::IsA"),
+                IsA(_) => imports.add("glib::prelude::*"),
```

Comment:
This one should not really be needed as `crate::prelude::*` should re-export it or not?

Reply:
We don't add crate::prelude::* anywhere yet

Reply:
Ok so this can get merged then for now, and other things are for other PRs? :)

Reply:
that would be my preference